src/CAREUI/misc/PrintPreview.tsx

```diff
@@ -14,31 +13,57 @@ type Props = {
   title: string;
 };
 
-export default function PrintPreview(props: Props) {
-  const normalScale = useBreakpoints({ default: 0.44, md: 1 });
+export default function PrintPreview({
+  children,
+  disabled,
+  className,
+  title,
+}: Props) {
   const { t } = useTranslation();
+  const [scale, setScale] = useState(1);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth < 640) {
+        setScale(0.44);
+      } else if (window.innerWidth < 1024) {
+        setScale(0.7);
+      } else {
+        setScale(1);
+      }
+    };
+
+    handleResize();
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
```

Comment:
Let's not do this. A mobile will not magically transform to a desktop or vice-versa.

Even if needed, this should be logic part of the `ZoomProvider` component and not `PrintPreview`.

If needed, create a separate issue and PR. Avoid any changes to this component unless absolutely necessary to solve the linked issue.

Reply:
@rithviknishad removed mobile, we can create another PR for that, check desktop version and ui